ABOUT.md

```diff
@@ -1,9 +1,106 @@
+ All of these are supported by a collection of general purpose methods for evaluation, parameter tuning, preprocessing, serialisation and I/O.
+ * Shogun has an **extensive testing infrastructure**, making it reliable on dozens of OS setups.
+
+## <a name="useshoguntolearn"></a> Using Shogun to Learn
+
+Shogun is a community learning platform in every sense: **anyone can use the toolbox to learn about ML** and apply it to solve problems.
+
+ * Many **scientists** use Shogun as a tool for their research and it is [cited many hundred times on Google scholar](http://scholar.google.com/scholar?hl=en&q=shogun+toolbox&btnG=&as_sdt=1%2C33&as_sdtp=).
+ * We offer the Shogun cloud for **educational** purposes at universities and workshops (e.g. University College London *Introduction to Machine Learning*).
+
+Even more important, **everyone can gain skills by helping us to build Shogun**. We highly value our contributors and want to foster and develop them.
+
+ * We have mentored 33 **Google Summer of Code student projects** since 2011 (and counting).
+ Especially female students and students from diverse backgrounds are encouraged to apply.
+ * Many of our alumni students took their acquired skills to industry and we are proud to be part of their education.
+
+## <a name="shogunhistory"></a> Shogun History
+
+Shogun got initiated in 1999 by [Soeren Sonnenburg](http://sonnenburgs.de/soeren) and [Gunnar Raetsch](http://www.raetschlab.org/) (that's where the name *ShoGun* originates from). 
+The original focus was on large-scale kernel methods and bioinformatics. 
+Since then, Shogun has continuously been used for scientific research (a list of papers citing Shogun can be found [here](http://scholar.google.com/scholar?hl=en&q=shogun+toolbox&btnG=&as_sdt=1%2C33&as_sdtp=)) and massively expanded.
+
+Shogun's features now span the whole space of Machine Learning methods:
+it includes many classic methods in classification, regression, dimensionality reduction and clustering, but also advanced algorithm classes such as metric, multi-task, structured output and online learning, as well as feature hashing, ensemble methods and optimization (to name just a few). 
+We aim to keep up with exclusive state-of-the art algorithms and have built a wealth of efficient SVM implementations, Multiple Kernel Learning, kernel hypothesis testing and Krylov methods, etc.
+In addition, all algorithms are now supported by a collection of general purpose methods for evaluation, parameter tuning, preprocessing, serialisation & I/O, etc. 
+ The resulting combinatorial possibilities are huge and you can check out some of them in our [showroom](http://shogun.ml/notebook/latest/).
+
+Currently, Shogun is developed by a diverse team of students, scientists and professionals (see [website](http://shogun-toolbox.org/page/about/ourteam) and [AUTHORS](https://github.com/shogun-toolbox/shogun/wiki/AUTHORS). 
```

Comment:
The links are still dead. Here are the correct ones:
https://github.com/shogun-toolbox/shogun/wiki/AUTHORS

The website "our team" thing doesnt exist anymore, just remove. 

"Currently, Shogun is developed by a diverse team of students, scientists and professionals see `[AUTHORS](https://github.com/shogun-toolbox/shogun/wiki/AUTHORS)`. "